nifi-frontend/src/main/frontend/apps/standard-content-viewer/src/app/pages/standard-content-viewer/feature/standard-content-viewer.component.scss

```diff
@@ -0,0 +1,26 @@
+/*!
+ *  Licensed to the Apache Software Foundation (ASF) under one or more
+ *  contributor license agreements.  See the NOTICE file distributed with
+ *  this work for additional information regarding copyright ownership.
+ *  The ASF licenses this file to You under the Apache License, Version 2.0
+ *  (the "License"); you may not use this file except in compliance with
+ *  the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+.standard-content-viewer {
+    h3 {
+        margin-bottom: 0;
+
+        .mime-type-display-name {
+            text-transform: uppercase;
```

Comment:
This seems like we should be using the tailwind 'uppercase' class. Is there a reason this h3 needs to be all upper case? It seems all of our other page header `h3`'s are not.

Reply:
Disregard this question. I see now that you are doing this for text like CSV or other mime type. I guess you could use angular in the template to conditionally apply the tailwind class for these strings.

Reply:
I've applied your suggestion below.